asttokens/mark_tokens.py

```diff
@@ -171,8 +173,9 @@ def handle_comp(self, open_brace, node, first_token, last_token):
     util.expect_token(before, token.OP, open_brace)
     return (before, last_token)
 
-  def visit_listcomp(self, node, first_token, last_token):
-    return self.handle_comp('[', node, first_token, last_token)
+  if sys.version_info[:2] < (3, 8):
```

Comment:
Do we really want to exclude this? If so, could you add a comment to explain? Also, isn't it enough to compare `sys.version_info < (3, 8)` (without taking a slice)?

Reply:
`col_offset` was changed for list comprehensions and tuples in 3.8: https://bugs.python.org/issue31241

Code to try it out:

```python
import ast

source = "[x for x in y]"
tree = ast.parse(source)
comp = tree.body[0].value
print(comp, comp.col_offset)
```